istioctl/pkg/install/verify.go

```diff
@@ -464,3 +467,10 @@ func allOperatorsInCluster(client dynamic.Interface) ([]*v1alpha1.IstioOperator,
 	}
 	return retval, nil
 }
+
+// gogoWorkaround prepares an unstructed for github.com/gogo/protobuf's Unmarshal().  That method cannot
+// unmarshal Kubernetes time.Time.
+func gogoWorkaround(un *unstructured.Unstructured) {
```

Comment:
Any specific reason to name it `gogoWorkaround`? The function doesn't do anything related to `gogo`. 
If I recall correctly,  @sdake was working on removing gogo related stuff.

Reply:
I call it that because it is a flaw in gogo's unmarshall, at least the last time I checked.